MTM1M3Test.py

```diff
@@ -568,7 +568,25 @@ def assertListAlmostEqual(self, l1, l2, msg=None, **kwargs):
         for i in range(len(l1)):
             self.assertAlmostEqual(l1[i], l2[i], msg=msg, **kwargs)
 
-    async def runActuators(self, function):
+    def get_enabled_force_actuators(self):
+        """Returns 156-member array, where true means actuator is enabled.
+
+        Returns
+        -------
+        enabled : `[bool]`
+            Array where true means actuator is enabled.
+        """
+        enabledFA = self.m1m3.evt_enabledForceActuators.get()
```

Comment:
Could you change `enabledFA` to `enabled_force_actuator`? I checked with Tiago and he told me we want this as a `snake_case`. 

We also want to use `.aget()` instead of `.get()` to avoid race conditions.

Reply:
I need get. aget will not return None if no data are available, it will wait for the data - see salobj doc.

Reply:
Case fixed.